packages/datadog-plugin-pg/test/index.spec.js

```diff
@@ -83,15 +94,26 @@ describe('Plugin', () => {
                 expect(traces[0][0].meta).to.have.property('db.name', 'postgres')
                 expect(traces[0][0].meta).to.have.property('db.user', 'postgres')
                 expect(traces[0][0].meta).to.have.property('db.type', 'postgres')
-                expect(traces[0][0].meta).to.have.property('span.kind', 'client')
```

Comment:
Same here, why is the test for `span.kind` removed?

Reply:
It's checked in the line 93 already.